projects/demo/src/modules/components/input-year/examples/1/index.html

```diff
@@ -1,10 +1,13 @@
-<tui-input-year
-    [formControl]="control"
-    [tuiTextfieldCleaner]="true"
->
-    Choose a year
+<tui-textfield #textfield>
     <input
-        placeholder="Not 2022 please"
-        tuiTextfieldLegacy
+        placeholder="Choose a year"
+        tuiInputYear
+        [(ngModel)]="value"
     />
-</tui-input-year>
+
+    <tui-calendar-year
+        *tuiTextfieldDropdown
+        [value]="value"
+        (yearClick)="textfield.handleOption($event)"
```

Comment:
This should happen internally inside the component, I think.

Reply:
fixed